app/pages/blackout/index.jsx

```diff
@@ -139,7 +141,8 @@ class BlackoutPage extends React.Component {
 
 function mapStateToProps(state) {
     return {
-        status: state.subscribeState
+        status: state.subscribeState,
+        theme: state.theme.data
```

Comment:
using the redux theme for styling. It will take the send the redux state's theme data as a prop named theme into the BlackoutPage component